packages/aws-client/tests/S3/test-multipartCopyObject.js

```diff
@@ -73,7 +73,7 @@ test('multipartCopyObject() copies a file between buckets', async (t) => {
     Key: sourceKey
   });
 
-  await multipartCopyObject({
+  const { ETag: etag } = await multipartCopyObject({
```

Comment:
🐜🐜🐜 Nit-picking question, but what is the benefit to renaming this variable in the test?

Reply:
No benefit. That's just me being pedantic with JS variable name casing. No other reason. (BTW, it bugs me how the AWS Node SDK doesn't use camel case conventions for object property names.)